_posts/2017-07-18-release-3-1-0.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Release 3.1.0"
+date: 2017-07-18 10:00:00
+author: "@Grummfy"
+categories: release
+---
+
+We are proud to announce the availability of atoum [3.1.0][milestone]!
+
+## Stats
+
+* 33 [commits][commits],
+* 277 files changed,
+* 7 contributors,
+* 7 [new features][changelog],
+* 1 [bug fix][changelog].
+
+## What's new
+
+Several new feature and a bug fix came with this release.
+
+Thing related to PHP version 7.1 & 7.2 to let atoum be ready for the next version of PHP:
+* The mock generator can now manage nullable type and handle the void type
+* The mock generator support strict types
+* The object asserter has been renamed phpObject to avoid issue related to reserved words in PHP 7.2 
+
+### Assertions & asserters
+
+#### Exception
+Now the asserter on exception has an `isInstanceOf` that can be used assertions without brackets.
```

Comment:
in fact I don't really see the inntrest of using isInstanceOf without brackets ... I just copy/paste the changelog, but this make no sense...